covid19-notebooks/covid19-notebook-etl-requirements.txt

```diff
@@ -4,3 +4,7 @@
 google-cloud-bigquery[bqstorage,pandas]>=2.13.1,<3.0.0
 jupyter>=1.0.0,<2.0.0
 seaborn>=0.11.1,<1.0.0
+matplotlib==3.4.3
+plotly==5.5.0
+scipy==1.7.2
+awscli==1.18.*
```

Comment:
we have 3 ways to install requirements now (directly in the dockerfile, in this requirements file and in the conda config file), and some libraries are installed more than one way, that doesn't sound like a good idea 😬 if [this comment](https://github.com/uc-cdis/covid19-tools/blob/2bb91c26e9caea68271907ebc4dd4045be706096/covid19-notebooks/covid19-notebook-etl-requirements.txt#L1-L2) is not accurate anymore we should move them out of the dockerfile. And if we could have all the requirements in one place that would be better. Not sure how much work that is...

Reply:
I removed the `covid19-notebook-etl-requirements.txt` file and put all the requirements in `covid19-notebook-etl-environment.yml` to have everything in one place. The image was built successfully without any problem. The `python3 sir_model.py` can run as well.😉